tests/application-test.js

```diff
@@ -486,6 +486,23 @@ describe('Box.Application', function() {
 			});
 		});
 
+		it('should still be called with a null element when an event occurs on a recently detached element', function() {
+			// Background on this edge case:
```

Comment:
Nice, can you also put this in getNearestElement() to explain why adding `element` to the condition is important?